src/Parsers/ASTAlterQuery.cpp

```diff
@@ -75,6 +82,53 @@ ASTPtr ASTAlterCommand::clone() const
     return res;
 }
 
+const char * ASTAlterCommand::typeToString(ASTAlterCommand::Type type)
+{
+    switch (type)
+    {
+        case ADD_COLUMN: return "ADD_COLUMN";
+        case DROP_COLUMN: return "DROP_COLUMN";
+        case MODIFY_COLUMN: return "MODIFY_COLUMN";
+        case COMMENT_COLUMN: return "COMMENT_COLUMN";
+        case RENAME_COLUMN: return "RENAME_COLUMN";
+        case MATERIALIZE_COLUMN: return "MATERIALIZE_COLUMN";
+        case MODIFY_ORDER_BY: return "MODIFY_ORDER_BY";
+        case MODIFY_SAMPLE_BY: return "MODIFY_SAMPLE_BY";
+        case MODIFY_TTL: return "MODIFY_TTL";
+        case MATERIALIZE_TTL: return "MATERIALIZE_TTL";
+        case MODIFY_SETTING: return "MODIFY_SETTING";
+        case RESET_SETTING: return "RESET_SETTING";
+        case MODIFY_QUERY: return "MODIFY_QUERY";
+        case REMOVE_TTL: return "REMOVE_TTL";
+        case REMOVE_SAMPLE_BY: return "REMOVE_SAMPLE_BY";
+        case ADD_INDEX: return "ADD_INDEX";
+        case DROP_INDEX: return "DROP_INDEX";
+        case MATERIALIZE_INDEX: return "MATERIALIZE_INDEX";
+        case ADD_CONSTRAINT: return "ADD_CONSTRAINT";
+        case DROP_CONSTRAINT: return "DROP_CONSTRAINT";
+        case ADD_PROJECTION: return "ADD_PROJECTION";
+        case DROP_PROJECTION: return "DROP_PROJECTION";
+        case MATERIALIZE_PROJECTION: return "MATERIALIZE_PROJECTION";
+        case DROP_PARTITION: return "DROP_PARTITION";
+        case DROP_DETACHED_PARTITION: return "DROP_DETACHED_PARTITION";
+        case ATTACH_PARTITION: return "ATTACH_PARTITION";
+        case MOVE_PARTITION: return "MOVE_PARTITION";
+        case REPLACE_PARTITION: return "REPLACE_PARTITION";
+        case FETCH_PARTITION: return "FETCH_PARTITION";
+        case FREEZE_PARTITION: return "FREEZE_PARTITION";
+        case FREEZE_ALL: return "FREEZE_ALL";
+        case UNFREEZE_PARTITION: return "UNFREEZE_PARTITION";
+        case UNFREEZE_ALL: return "UNFREEZE_ALL";
+        case DELETE: return "DELETE";
+        case UPDATE: return "UPDATE";
+        case NO_TYPE: return "NO_TYPE";
+        case LIVE_VIEW_REFRESH: return "LIVE_VIEW_REFRESH";
+        case MODIFY_DATABASE_SETTING: return "MODIFY_DATABASE_SETTING";
+        case MODIFY_COMMENT: return "MODIFY_COMMENT";
```

Comment:
I think it is easier to use `enum_name` function in contrib/magic_enum/include/magic_enum.hpp to get the  name of enum value

Reply:
For system queries I prefer not to use enum_name, because it returns temporary string, but we return const char *, and we need to be sure that enum name will be allocated statically. We can create lookup table check `ASTSystemQuery`, but for simplicity we can just handle cases manually.